### PR TITLE
 Fix CWE parsing for SARIF parser (#4940)

### PR DESCRIPTION
**Pull request by akitibala**
Fix:#4940
SARIF findings import Error before fix :
sarif_error

after fix SARIF imports findings successfully :

sarif_import_success

file used for importing:
[sast_mobile_mobsfscan.zip](https://github.com/DefectDojo/django-DefectDojo/files/6991924/sast_mobile_mobsfscan.zip)

### CODE REVIEW

**Comment by valentijnscholten** [comment]
Thanks, could you add the mobscan to the unit tests as "proof" and to prevent future regressions? And add some asserts to the unit tests.

File: .gitignore [hunk 1]
```diff
@@ -133,3 +133,4 @@ helm/defectdojo/charts
 docs/public
 docs/node_modules
 docs/resources
+.pre-commit-config.yaml
```
Comment (StefanFl) [inline .gitignore:136]: This additional line in .gitignore seems not to be part of the bugfix. Please remove it.

**Review by StefanFl — DISMISSED** [review]

File: .gitignore (hunk 1 above)
Comment (akitibala) [reply .gitignore:136]: sure

**Comment by akitibala** [comment]
> 
> 
> Thanks, could you add the mobscan to the unit tests as "proof" and to prevent future regressions? And add some asserts to the unit tests.

**Review by Maffooch — APPROVED** [review]

**Review by valentijnscholten — APPROVED** [review]

**Comment by valentijnscholten** [comment]
dismissed your review @StefanFl as it was fixed and wanted to merge :-)

**Pull request merged**